presto-benchmark-driver/src/main/java/com/facebook/presto/benchmark/driver/RegexTemplate.java

```diff
@@ -38,7 +37,7 @@
             NAMED_GROUPS_METHOD.setAccessible(true);
         }
         catch (NoSuchMethodException e) {
-            throw Throwables.propagate(e);
+            throw new AssertionError(e);
```

Comment:
Why `AssertionError`?

Reply:
It seems reasonable since this is a compile time bug (wrong method used) or something changed in the JVM, etc. This is a "should never happen" situation and it's not recoverable.

But if you feel that this should be a different exception, I'm fine to change it.

Reply:
No, just curious. That makes sense.